hashintel/hash
# Provide `onCellEdited` with sorted & flattened row data

## 🌟 What is the purpose of this PR?

This PR fixes a bug caused by "moving row sorting logic into `grid.tsx`". It caused `onCellEdited` to work with unsorted & un-flattened data. Which broke the entity editor update property functionality. Sort of the updated row was different then the sort of the table.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203301853897838/1203433145282329/f) _(internal)__

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open entity editor
1. Updating property values should work as expected
